hal/src/rtl872x/gpio_hal.cpp

```diff
@@ -68,6 +68,10 @@ bool isCachePin(hal_pin_t pin) {
     return false;
 }
 
+bool isVbatMeasPin(hal_pin_t pin) {

```

Comment:
Perhaps generalize this to something like `isGpioPin()` and check for `!= RTL_PORT_NONE`?

Reply:
Updated.